app/controllers/concerns/secured.rb

```diff
@@ -27,8 +28,11 @@ def auth_token
     JsonWebToken.verify(http_token)
   end
 
-  def get_current_user!
+  def get_auth0_id
     @auth0_id = decode_token!
```

Comment:
Need to make this a separate method because `POST /users` needs the `auth0_id` but does not have a user yet